alws/build_planner.py

```diff
@@ -280,21 +280,19 @@ async def add_task(self, task: build_schema.BuildTaskRef):
             )
             if task.module_version:
                 module_version = int(task.module_version)
+            mock_enabled_modules = mock_options.get('module_enable', [])[:]
+            # Take first task mock_options becouse it duplicates on each tasks
```

Comment:
I'd better say: "We take the first task mock_options as all tasks share the same mock_options"

Reply:
Also, the comment could be moved into line 286, above ```mock_enabled_modules.extend(```